test/integration/runner_task_test.rb

```diff
@@ -284,6 +284,24 @@ def test_run_adds_env_vars_provided_to_the_task_container
     ], in_order: true)
   end
 
+  def test_run_adds_custom_image_provided_to_the_task_container
+    deploy_task_template
+
+    task_runner = build_task_runner
+    result = task_runner.run(
+      template: 'hello-cloud-template-runner',
+      command: ['/bin/sh', '-c'],
+      arguments: ['echo "The value is: $MY_CUSTOM_VARIABLE"'],
```

Comment:
Does this do the right thing w/o actually setting a env_var called `MY_CUSTOM_VARIABLE` ?

Reply:
Good point. I'll remove the `env_var` part from `arguments`